File: lista 2 casa C/questao 2.c
#include <stdio.h>

int main(){
    int numero_c=20;
    int N;
    int R;
    printf("Digite um numero entre 0 e 100 para ver o quao distante ele esta do numero chave:");
    scanf("%d", &N);
    R = N - numero_c;
    if (N < numero_c){
        printf("o numero digitado é menor que o numero chave");
    } else{
        printf("o numero digitado esta %d de distancia do numero chave", R);
    }
}

File: listas de exercicio/listas 2/lista 2 casa python/questao 2.py
NC = 15
NC = float(NC)
A = input('Digite um valor positivo entre 0 e 100 para saber quao distante ele esta do numero chave:')
A = float(A)
R = A - NC
if A > 100:
    input("erro numero digitado maior que 100")
elif R < 0:
    input(f"erro {R} é um numero negativo")
else:
    input(f"ele está {R} distante")
